### DISCUSSION TITLE
How to print stdout when walker exists with plugins

### DISCUSSION THREAD

**Discussion by omnigenous** [abenz1267/walker#60] [Q&A]
Somewhat related to #59

I have tried:

```json
{
  "name": "test",
  "prefix": "t",
  "src": "echo \"c\nb\na\"",
  "cmd": "cat"
}
```

**Comment by abenz1267** [comment]
Not sure what you mean.

The problem is: if you run Walker as a service, starting Walker will just tell the service to display the window. The processing is happening in the service-instance, not the one you opened it with. So any prints will happen in the main instance.

Even then: you have a running process, Walker, "cat" would imply printing to stdout... but Walker is the running process, not the shell. So Walker would need to get the return of `cat` and print that itself.

Sounds weird. Not sure what the use-case here is.

**Reply by omnigenous** [reply]
I need to run few experiments and do some reading then I can try to explain my use-case once again if it still relevant.
